app/components/location_request_link_component.rb

```diff
@@ -64,9 +60,15 @@ def external_link_icon
   end
 
   def link_text
+    return I18n.t('searchworks.request_link.finding_aid') if finding_aid?
```

Comment:
Isn't it possible there are records with findings aids that are not in a library that's doing Aeon?

Reply:
It's definitely possible...would it make sense to move the check for "is it in an Aeon library" here too? I feel like logic is kinda bleeding over from requests into here